rules/windows/persistence_scheduled_task_updated.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2022/08/29"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/08/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Indicates the update of a scheduled task using Windows event logs. Adversaries can use these to establish persistence,
+by changing the configuration of a legit scheduled task. Some changes such as disabling or enabling a scheduled task are
+common and may may generate noise.
+"""
+false_positives = ["Legitimate scheduled tasks may be created during installation of new software."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "A scheduled task was updated"
+references = ["https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4698"]
+risk_score = 21
+rule_id = "a02cb68e-7c93-48d1-93b2-2c39023308eb"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+iam where event.action == "scheduled-task-updated" and
+
+ /* excluding tasks created by the computer account */
+ not user.name : "*$" and
+ not winlog.event_data.TaskName :
```

Comment:
This could be really noisy. Why exclude these - seems random? I'm sure it is based on observed noise, so what is the intent of the rule long term for noisy tasks like these?

Reply:
for now the 4698 event task config details (cmd to execute, trigger etc.) are not parsed, so unfortunately we can't use it for blacklist based detection (e.g. task body contains rundll32, powershell, mshta and rest), so only way to whitelist now is user name and task name (fragile, thus the broad aspect of the rule). 4698 is rarely enabled in prod env (need audit other setting to be enabled), if so I don't expect the noise to be too high and if so we can exclude top noisy task names as a minimum. its an important event/technique to cover broadly.